DTS-STN/canadian-dental-care-plan
# Removing `communicationPreference` from renewal flow state

### Description
When attempting to map the renew adult/child state to renewal DTO, I noticed that the `communicationPreference` state isn't being used anywhere, so all references to this state in the renewal flow have been removed.

Therefore, when submitting a renewal application, the `communicationPreference` will be what's on file for the applicant (will be done in a future PR).

### Checklist

### Test Instructions
Run application and navigate to `/en/renew`. Complete the renewal flow and submit a renewal application. It should still work.
